src/App.css

```diff
@@ -0,0 +1,14 @@
+.App {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  height: 100vh;
+}
```

Comment:
The contents of the `App` are centered which affects the position of the navigation component.
Below is a screenshot.
![app_centered](https://user-images.githubusercontent.com/43215910/186134578-00588875-54c8-48bc-bd56-a70fffefb64d.png)

I would recommend removing the `justify-content center` from the `App` class.